maintenance-site/index.html

```diff
@@ -126,6 +126,26 @@ <h1>App en maintenance</h1>
     </svg>
   </div>
   <script>
+    const allowedDomains = [
+      'https://web.testing.passculture.team',
+      'https://app.testing.passculture.team',
+      'https://pro.testing.passculture.team',
+      'https://backend.testing.passculture.team',
+      'https://web.staging.passculture.team ',
+      'https://app.staging.passculture.team',
+      'https://pro.staging.passculture.team',
+      'https://backend.staging.passculture.team',
+      'https://web.integration.passculture.app',
+      'https://integration.passculture.app',
+      'https://backend.integration.passculture.app',
+      'https://integration.passculture.pro',
+      'https://backend.integration.passculture.pro',
+      'https://web.passculture.app',
+      'https://passculture.app',
+      'https://backend.passculture.app',
+      'https://passculture.pro',
+      'https://backend.passculture.pro',
```

Comment:
À ma connaissance, cette page n'est utilisée que depuis la webapp et le portail pro, donc on ne devrait pas avoir besoin d'inclure les URLs du backend, je pense ? Seulement les URLs des webapp (v1 et peut-être v2) et portail pro.

```suggestion
      'https://web.testing.passculture.team',
      'https://app.testing.passculture.team',
      'https://pro.testing.passculture.team',
      'https://web.staging.passculture.team ',
      'https://app.staging.passculture.team',
      'https://pro.staging.passculture.team',
      'https://web.integration.passculture.app',
      'https://integration.passculture.app',
      'https://integration.passculture.pro',
      'https://web.passculture.app',
      'https://passculture.app',
      'https://passculture.pro',
```

Reply:
Je me suis dit que c'était nos domaines donc si jamais un jour le back peut gérer une 302 ou retourner du html, avec une source, alors ça peut être pratique.